Sources/DeepLearning/Layer.swift

```diff
@@ -51,6 +51,103 @@ public extension Layer
     }
 }
 
+/// A sequencing of two Layers to form a chain.
+///
+/// The sequential combinator composes two layers sequentially, feeding the
+/// output of the first layer as input to the second layer.
+///
+/// It is normally constructed via the >> operator.
+public struct Sequential<LHS: Layer, RHS: Layer>: Layer
```

Comment:
I'd suggest renaming `LHS` and `RHS` (along with their corresponding properties, `lhs` and `rhs`) to unabbreviated words, e.g. `First` and `Second`.